packages/instant/webpack.config.js

```diff
@@ -135,6 +135,9 @@ const generateConfig = (dischargeTarget, heapConfigOptions, rollbarConfigOptions
         resolve: {
             extensions: ['.js', '.json', '.ts', '.tsx'],
         },
+        node: {
+            fs: 'empty'
+          },
```

Comment:
the building of instant throw error on ed25519.js package and this line help to ignore this package while building